agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.cpp

```diff
@@ -2105,11 +2119,18 @@ bool backhaul_manager::handle_client_capability_query(ieee1905_1::CmduMessageRx
     //with any of the BSS operated by the Multi-AP Agent [ though the TLV does contain a BSSID, the specification
     // says that we should answer if the client is associated with any BSS on this agent.]
     bool associated_client_found = false;
+    sClientInfo s;
+    // sMacAddr client_ruid;
+    // sMacAddr client_vap;
     for (const auto &slave : m_radio_info_map) {
         auto associated_clients_map = slave.second.associated_clients_map;
```

Comment:
```suggestion
        const auto &associated_clients_map = slave.second.associated_clients_map;
```

This was already wrong before this PR, but I notice it now...